Octokit/Clients/MiscellaneousClient.cs

```diff
@@ -3,9 +3,8 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Threading.Tasks;
-#if NET_45
 using System.Collections.ObjectModel;
-#endif
+
```

Comment:
extra whitespace line (there are a few similar occurences on other files).  this is definitely sraping the bottom of the barrel in terms of what I can pick up in review!